.github/workflows/openhands-resolver.yml

```diff
@@ -106,20 +110,34 @@ jobs:
 
       - name: Check required environment variables
         env:
-          LLM_MODEL: ${{ secrets.LLM_MODEL }}
+          LLM_MODEL: ${{ secrets.LLM_MODEL || inputs.LLM_MODEL }}
           LLM_API_KEY: ${{ secrets.LLM_API_KEY }}
           LLM_BASE_URL: ${{ secrets.LLM_BASE_URL }}
           PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
           PAT_USERNAME: ${{ secrets.PAT_USERNAME }}
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
Sorry, this is probably obvious and I'm missing something, could you please tell where or how is this `github.token` set to be available to the workflow here?

Reply:
It is added by github itself.
https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication

Reply:
Ah, thanks!